### ISSUE TITLE
Module '"@firebase/app"' has no exported member '_FirebaseAppInternal'.

### ISSUE THREAD

**Issue by CoooWeee** [firebase/firebase-js-sdk#5382]
### Describe your environment

  * Operating System version: osx 11.55.2
  * Browser version:  chrome 92.0.4515.159
  * Firebase SDK version: 9
  * Firebase Product: app-compat (auth, database, storage, etc)

#### Steps to reproduce:

Upgrade from Firebase v8 to v9
use exp angular/fire and import AngularFireModule from @angular/fire/compat and try to build using ng build

#### Relevant Code:

```javascript
Error: node_modules/@firebase/app-compat/dist/src/firebaseApp.d.ts:19:10 - error TS2305: Module '"@firebase/app"' has no exported member '_FirebaseAppInternal'.

19 import { _FirebaseAppInternal as _FirebaseAppExp } from '@firebase/app';
```

[as reported here](https://github.com/angular/angularfire/issues/2853)

**Comment by petrusgomes** [comment]
For those that don't wan't to watch the video, the solution from @edocbuhtig is to add:

`"skipLibCheck": true`

to `tsconfig.json -> "angularCompilerOptions"`

**Comment by Jbz797** [comment]
And for users aren't on Typescript ?

**Comment by Feiyang1** [comment]
This has been fixed in `9.0.1`.

@Jbz797 there is no impact on Javascript users.
